Sources/Logger/Level.swift

```diff
@@ -8,12 +8,68 @@
 import Foundation
 
 /// Enum representing different possible levels for log messages.
-public enum Level: String, CaseIterable {
-    case error = "ERROR"
-    case warn = "WARNING"
-    case info = "INFO"
-    case debug = "DEBUG"
-    case verbose = "VERBOSE"
-    case system = "SYSTEM"
-    case process = "PROCESS"
+public enum Level {
+    case error
+    case warn
+    case info
+    case debug
+    case verbose
+    case system
+    case process
+    case custom(CustomStringConvertible)
```

Comment:
Vidim potencial pro vlastni loggery, ale chce to minimalne nahradit allCases s něčím podobným. Třeba standardCases, které vrati všechno kromě custom.

Reply:
jj souhlas, pridal sem `standardCases`.